torchvision/datasets/mnist.py

```diff
@@ -169,14 +199,24 @@ class FashionMNIST(MNIST):
         target_transform (callable, optional): A function/transform that takes in the
             target and transforms it.
     """
+    training_file = 'fashion-mnist-training.pt'
+    test_file = 'fashion-mnist-test.pt'
+
     urls = [
         'http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/train-images-idx3-ubyte.gz',
         'http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/train-labels-idx1-ubyte.gz',
         'http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/t10k-images-idx3-ubyte.gz',
         'http://fashion-mnist.s3-website.eu-central-1.amazonaws.com/t10k-labels-idx1-ubyte.gz',
     ]
-    classes = ['T-shirt/top', 'Trouser', 'Pullover', 'Dress', 'Coat', 'Sandal',
-               'Shirt', 'Sneaker', 'Bag', 'Ankle boot']
+
+    md5s = {
+        't10k-images-idx3-ubyte.gz': 'bef4ecab320f06d8554ea6380940ec79',
+        't10k-labels-idx1-ubyte.gz': 'bb300cfdad3c16e7a12a480ee83cd310',
+        'train-images-idx3-ubyte.gz': '8d4fb7e6c68d591d4c3dfef9ec88bf0d',
+        'train-labels-idx1-ubyte.gz': '25c81989df183df01b3e8a0aad5dffbe',
+    }
+
+
```

Comment:
Lint is complaining because there are too many empty lines
```
./torchvision/datasets/mnist.py:222:1: E303 too many blank lines (4)
```
Could you fix this?